manager/src/main/java/com/redhat/service/smartevents/manager/dns/KnativeBrokerPathBuilder.java

```diff
@@ -0,0 +1,9 @@
+package com.redhat.service.smartevents.manager.dns;
+
+// At the moment we have to craft the endpoint as the fleet-shard/knative would do.
+// Remove this logic when https://github.com/knative/eventing/issues/6467 is fixed and we can resolve the path directly in the Ingress/Route
+public class KnativeBrokerPathBuilder {
```

Comment:
@r00ta Do we have a JIRA to make sure we don't forget to remove this? 

cheers,
Rob